jujugui/static/gui/src/app/app.js

```diff
@@ -1775,31 +1799,33 @@ YUI.add('juju-gui', function(Y) {
         }
         return;
       }
-      var credentials = this.env.getCredentials();
       // After re-arranging the execution order of our routes to support the
       // new :gui: namespace we were unable to log out on prod build in Ubuntu
       // chrome. It appeared to be because credentials was null so the log in
       // form was never shown - this handles that edge case.
-      var noCredentials = !(credentials && credentials.areAvailable);
-      if (noCredentials) {
-        this.set('loggedIn', false);
-        // If there are no stored credentials redirect to the login page
-        var component = this.state.getState('current', 'app', 'component');
-        if (!component && component !== 'login') {
-          this.state.dispatch({
-            app: {
-              component: 'login',
-              metadata: {
-                redirectPath: this.get('currentUrl')
-              }
-            }
-          });
+      if (this.env.getCredentials().areAvailable) {
```

Comment:
While unlikely, if the localStorage call fails then the value returned back from `getCredentials` won't be an object so this will fall over.